colossus-examples/src/main/scala/colossus-examples/WebsocketExample.scala

```diff
@@ -52,10 +52,10 @@ object WebsocketExample {
     
     val generator = io.actorSystem.actorOf(Props[PrimeGenerator])
 
-    Server.basic("websocket", port){ new Service[Http](_) {
+    Server.basic("websocket", port){ ctx => new Service[Http](ctx) {
       def handle = {
         case UpgradeRequest(resp) => {
-          become(new WebsocketHandler(_) with ProxyActor {
+          become(_ => new WebsocketServerHandler(ctx) with ProxyActor {
```

Comment:
I've created #359 to address the fact that the input context is now not used here.